app/src/main/java/com/example/trainingintive/util/ErrorMessageId.kt

```diff
@@ -0,0 +1,16 @@
+package com.example.trainingintive.util
+
+import com.example.trainingintive.R
+import java.lang.Exception
+import java.net.UnknownHostException
+
+class ErrorMessageId {
```

Comment:
In such cases (when you don't want to create an instance of the class to use it) simply use `object` instead of `class`. You can also store functions in a file outside of any class/object when you only need some util function.